source/locale/pt_BR/LC_MESSAGES/components-and-apis.po

```diff
@@ -175,16 +176,18 @@ msgid "`Auth  doc.`_"
 msgstr ""
 
 #: ../../source/components-and-apis.rst:53
-#: ../../source/components-and-apis.rst:157
-#: ../../source/components-and-apis.rst:162
+#: ../../source/components-and-apis.rst:161
+#: ../../source/components-and-apis.rst:166
 #: ../../source/components-and-apis.rst:167
-#: ../../source/components-and-apis.rst:172
-msgid "`API - auth`_"
+#: ../../source/components-and-apis.rst:171
+#: ../../source/components-and-apis.rst:176
+msgid
+ "`API - auth`_"
```

Comment:
shouldn't that be on the same line? If it was generated automatically, then ok...